assets/js/sidebar.js

```diff
@@ -49,7 +62,9 @@ function openSidebar (immediate) {
     '-ms-flex-basis': width,
     'flex-basis': width,
     width: width
-  }, duration)
+  }, duration, function () {
+    sidebar.attr('style', '')
```

Comment:
I believe I put this line here because the styles of the animation get set inline, so they override the CC media queries. When the animation completes, this line removes the inline styles and lets the CSS take over.

If you take this line away, and then adjust the size of your screen, the sidebar width won't adjust dynamically. It's probably easier to explain by just commenting out the line and trying resizing your browser window and playing with the sidebar opening and closing.